www/docs/intro.md

```diff
@@ -376,6 +376,14 @@ Devnet offers [empty block creation](https://shard-labs.github.io/starknet-devne
 const emptyBlock = await starknet.devnet.createBlock();
 ```
 
+#### Mint tokens to an account
+
+DevNet allows [minting token](https://shard-labs.github.io/starknet-devnet/docs/guide/mint-token#mint-with-a-transaction). You can call `starknet.devnet.mint` like this,
+
+```typescript
+await starknet.devnet.mint(account_address, 2e12, true);
```

Comment:
Would be worth noting what the last boolean parameter refers to

Reply:
Yeap! Sure would!